integration_tests/tests/features/Chat.feature

```diff
@@ -0,0 +1,43 @@
+# Copyright 2023 The Tari Project
+# SPDX-License-Identifier: BSD-3-Clause
+
+Feature: Chat messaging
+
+  Scenario: A message is propagated between nodes
+    Given I have a seed node SEED_A
+    When I have a chat client CHAT_A connected to seed node SEED_A
+    When I have a chat client CHAT_B connected to seed node SEED_A
+    When I use CHAT_A to send a message 'Hey there' to CHAT_B
+    Then CHAT_B will have 1 message with CHAT_A
+
+  Scenario: A message is sent directly between nodes
+    Given I have a seed node SEED_A
+    When I have a chat client CHAT_A connected to seed node SEED_A
+    When I have a chat client CHAT_B connected to seed node SEED_A
+    When I add CHAT_B as a contact to CHAT_A
+    When CHAT_A waits for contact CHAT_B to be online
```

Comment:
If the base node is required to establish comms between the chat clients, then it should be stopped after the chat clients' online status is confirmed.

Reply:
Yes, stopping the base node and having the clients communicate does work, I'll update the test.